backend/chalice/api_server/README.md

```diff
@@ -27,9 +26,17 @@ Filename                  | Purpose                           | Information link
 1.  Install dependencies
 
      ```shell
-     pip install -r requirements-dev.txt
+     pip install -r requirements.txt
      ```
 
+1.  Enable local connection to the private `dev` RDS instance:
+
+     ```shell
+     ssh -L 5433:corpora-dev-corpora-api.cluster-c81u9imopfwl.us-west-2.rds.amazonaws.com:5432 bastion.dev.single-cell.czi.technology
+     ```
+
+    Note: This command creates an ssh tunnel from `localhost:5433` to the RDS connection endpoint via a `bastion` server.
```

Comment:
There was a whole shebang around this wasn't there when I tried to connect? I ended up needing to install blessclient?

Reply:
Yeah, the first step in this section points to a "Pre-requisites" section that covers setting up `blessclient`. Sufficient?

Reply:
Oh oops! I missed that; yep, that works, thanks!